fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/LoanEvent.java

```diff
@@ -49,5 +49,6 @@ public enum LoanEvent {
     LOAN_REJECT_TRANSFER, //
     LOAN_WITHDRAW_TRANSFER, //
     LOAN_CREDIT_BALANCE_REFUND, //
-    LOAN_CHARGEBACK;
+    LOAN_CHARGEBACK, //
+    Loan_CHARGE_ADDED;
```

Comment:
Enum naming is not according to the standards, should be LOAN_CHARGE_ADDED.

Reply:
Nice catch!